src/android/UsabillaCordova.java

```diff
@@ -171,4 +208,13 @@ private void getDefaultDataMasks() {
             e.printStackTrace();
         }
     }
+
+    private void setUpBroadcastReceivers() {
+        receiverCampaignClosed = new BroadcastReceiver() {
+            @Override
+            public void onReceive(Context context, Intent intent) {
+                callbackContext.success(getResult(intent, FeedbackResult.INTENT_FEEDBACK_RESULT_CAMPAIGN));
```

Comment:
Can we update it like this 
JSONObject resultData = new JSONObject();
                    try {
                        resultData.put("result", getResult(data, FeedbackResult.INTENT_FEEDBACK_RESULT_CAMPAIGN));
                        result.put("completed", resultData);
                        callbackContext.success(result);
                    } catch (JSONException e) {
                        e.printStackTrace();
}